### ISSUE TITLE
[compilation] AppVeyor integration/badge

### ISSUE THREAD

**Issue by mamontov-cpp** [mamontov-cpp/saddy-graphics-engine-2d#134]

**Comment by mamontov-cpp** [comment]
Should we just append IrrKlang parts to be easily integrated with engine?

**Comment by mamontov-cpp** [comment]
https://github.com/simbody/simbody/blob/appveyor-mingw/appveyor.yml

**Comment by mamontov-cpp** [comment]
We could append irrklang parts, in fact, some repos already done this.

**Comment by mamontov-cpp** [comment]
https://github.com/libgd/libgd/blob/master/appveyor.yml

**Comment by mamontov-cpp** [comment]
[![AppVeyor](https://ci.appveyor.com/api/projects/status/u99u4tsb9dv6qwx3?svg=true)](https://ci.appveyor.com/project/mamontov-cpp/saddy-graphics-engine-2d)

**Comment by mamontov-cpp** [comment]
Further work is blocked by #142, and suspended until it will be fixed.

**Comment by mamontov-cpp** [comment]
We should implement option to disable qt-based builds in AppVeyor, since AppVeyor is bad at it.

**Comment by mamontov-cpp** [comment]
Started working on this issue.

**Comment by mamontov-cpp** [comment]
It seems there is a solution for issues with MinGW in https://github.com/jrfonseca/drmingw/blob/master/appveyor.yml

**Comment by mamontov-cpp** [comment]
https://appveyor-matrix-badges.herokuapp.com/repos/mamontov-cpp/saddy-graphics-engine-2d/branch/master/1

https://appveyor-matrix-badges.herokuapp.com/repos/mamontov-cpp/saddy-graphics-engine-2d/branch/master/2

**Comment by mamontov-cpp** [comment]
Finally done this.
